src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let result = 0;
+
+  for (let i = 1; i <= days; i++) {
```

Comment:
you can do without a loop in this task, just create variable

Reply:
Done